lte/gateway/release/build-magma.sh

```diff
@@ -79,6 +79,8 @@ case $BUILD_TYPE in
 esac
 
 
+sudo add-apt-repository 'deb [arch=amd64] https://dl.bintray.com/tetrate/getenvoy-deb stretch stable'
```

Comment:
This is supposed to be build only inside the vagrant VM and we already have those sources in it. Remove it

Reply:
oke